solidity/test/bridge/Bridge.Parameters.test.ts

```diff
@@ -234,12 +234,13 @@ describe("Bridge - Parameters", () => {
   })
 
   describe("updateRedemptionParameters", () => {
-    context("when caller is the contract guvnor", () => {
+    context("when caller is the contract governance", () => {
```

Comment:
I think this was [intentional](https://dictionary.cambridge.org/dictionary/english/guvnor), see that the rest of tests in this file use "guvnor" so it's not about a single typo.